frontend/src/components/AmountInput.tsx

```diff
@@ -0,0 +1,113 @@
+import { useQuery } from "@tanstack/react-query";
+import prettyPrintAmount from "@util/prettyPrintAmount";
+import { Dispatch, SetStateAction, useContext, useState } from "react";
+import { inputStyle, selectStyle } from "../styles";
+import { NodeManagerContext } from "./GlobalStateProvider";
+
+type Props = {
+    amountSats: string,
+    setAmount: Dispatch<SetStateAction<string>>,
+    accent: "green" | "blue" | "red",
+    placeholder: string,
+}
+
+export enum Currency {
+    USD = "USD",
+    SATS = "SATS"
+}
+
+function satsToUsd(amount: number, price: number): string {
+    let btc = amount / 100000000;
+    let usd = btc * price;
+    return usd.toFixed(2);
+}
+
+function usdToSats(amount: number, price: number): string {
+    let btc = amount / price;
+    let sats = btc * 100000000;
+    return sats.toFixed(0);
```

Comment:
we have a function to do btc <-> sats conversion in rust, not sure if we want to do that here instead?

Reply:
oh right forgot about that one